libraries/AP_RangeFinder/AP_RangeFinder.h

```diff
@@ -170,6 +170,7 @@ class RangeFinder
     
     // methods to return a distance on a particular orientation from
     // any sensor which can current supply it
+    float distance_m_orient(enum Rotation orientation) const;
     uint16_t distance_cm_orient(enum Rotation orientation) const;
```

Comment:
I think we should convert all methods over to meters, its a little odd having the distance in meters and the min and max in CM.

Reply:
The min and max are in cm because that's how they are used in parameters. These values are ints `RNGFND1_MIN_CM` and `RNGFND1_MAX_CM`. I prefer it this way because it just passes the parameters through